<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterCliente</class>
 <widget class="QMainWindow" name="MaterCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaterCliente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCadastrar">
       <attribute name="title">
        <string>Cadastrar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="rbclienteMode">
          <item>
           <widget class="QRadioButton" name="rbClienteFisico">
            <property name="text">
             <string>Cliente Físico</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbClienteJuridico">
            <property name="text">
             <string>Cliente Jurídico</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="dados">
          <item>
           <layout class="QVBoxLayout" name="lDados">
            <item>
             <widget class="QLabel" name="lblLogradouro">
              <property name="text">
               <string>Logradouro</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblSetor">
              <property name="text">
               <string>Setor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblCidade">
              <property name="text">
               <string>Cidade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblEstado">
              <property name="text">
               <string>Estado</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTelefone">
              <property name="text">
               <string>Telefone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblEmail">
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblCpf">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cpf</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblNome">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblCelular">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Celular</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblCnpj">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cnpj</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblRazaoSocial">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Razão Social</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblNomeContato">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Nome Contato</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="leDados">
            <item>
             <widget class="QLineEdit" name="leLogradouro">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leSetor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCidade">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leEstado">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leTelefone">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leEmail">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCpf">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNome">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCelular">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCnpj">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leRazaoSocial">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNomeContato">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnCadastrar">
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConsultar">
       <attribute name="title">
        <string>Consultar</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabListar">
       <attribute name="title">
        <string>Listar</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabRemover">
       <attribute name="title">
        <string>Remover</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
